src/transformStateWithClones.js

```diff
@@ -6,8 +6,39 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resultState = [];
```

Comment:
I believe in good naming, so we should make it maximally simple to read. What we return is not actually resultState, it is an array of all state versions. So we may do it like that:
```suggestion
  const states = [];
```
or if we want even better then:
```suggestion
  const stateVersions = [];
```